test/ERC721A.test.js

```diff
@@ -225,6 +235,13 @@ const createTestSuite = ({ contract, constructorArgs }) =>
             it('adjusts owners balances', async function () {
               expect(await this.erc721a.balanceOf(this.from)).to.be.equal(1);
             });
+
+            it('startTimestamp updated correctly', async function () {
+              expect(this.timestampBefore).to.be.lt(this.timestampToMine);
+              expect(this.timestampAfter).to.be.gte(this.timestampToMine);
+              expect(this.timestampAfter).to.be.gt(this.timestampBefore);
```

Comment:
Isn't this statement redundant given the first two checks?

Reply:
Oh yes, removed.